syncer/syncerservice_test.go

```diff
@@ -10,7 +11,14 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+func TestMain(m *testing.M) {
+	os.Exit(m.Run())
+}
```

Comment:
What does this code do?

Reply:
It's a sign of incomplete cleanup. I initially tried to change the log level because the test was taking a lot of meaningless logs, but I gave up because it couldn't be done easily. I reverted that code and missed it. 

I'll fix those you said next week.